docs/release-notes/software-releases.md

```diff
@@ -19,6 +19,30 @@ None to report.
 None to report.
 ------>
 
+## February 7, 2022: File Entity Page and Open Access Variant Filtering​
+
+Data Platform - 1.105.3 - API 3.28.2
+
+#### New Features
+
+- A new **File Entity** page has been released that displays the metadata about a file. This page is accessible from the [File Repository](https://platform.icgc-argo.org/repository) by clicking on a File ID. This page includes:
+  - A file summary with a link to all files for the associated program.
+  - Data analysis Information, that include helpful links to documentation about the associated ARGO DNA pipeline workflow and version.
+  - Some metadata about the associated donor, samples, and specimens for the file.
+  - The ability to download a manifest for the file.
+- The **Program Dashboard** has been updated to prepare for the [Open Access Variant Filtering workflow](/docs/analysis-workflows/dna-open-access-filtering):
+  - Once a program has raw reads that are processing in the DNA pipeline, the status of the Open Access VF workflow will be reported on the donor summary table and in the molecular chart (if that donor meets the tumour/normal pair requirements).
+- **File Repository** improvements have been made, which include:
+  - Adding a more user friendly File ID column to the table, that links to the file entity page.
```

Comment:
@ciaranschutte can you change this to 
`Adding the File ID column to the table, that links to the file entity page.`